signals/sender/sender_legitimate_ups_domain.yml

```diff
@@ -0,0 +1,4 @@
+name: "Sender: Legitimate UPS domain"
+type: "query"
+source: |  
+  sender.email.domain.root_domain in ("ups.com", "upsemail.com")
```

Comment:
May want to add a DMARC check here, unless risk score will weigh a DMARC failure more/a failure would be accounted for elsewhere.

Reply:
Yeah we have signals already for DMARC failures